tests/api_test.py

```diff
@@ -5018,6 +5032,23 @@ def foo(x):
     expected = 2. * jnp.cos(jnp.arange(3.))
     self.assertAllClose(ans, expected, check_dtypes=False)
 
+  def test_initial_style_vmap_with_collective(self):
+
+    @api.custom_vjp
```

Comment:
Is `custom_vjp` always initial style? I thought it only becomes that after tracing

Reply:
Right, I wrapped it in a eval_jaxpr(make_jaxpr(f)).